libraries/AP_RangeFinder/AP_RangeFinder_MAVLink.cpp

```diff
@@ -45,20 +45,27 @@ bool AP_RangeFinder_MAVLink::detect()
 
 /*
    Set the distance based on a MAVLINK message
-*/
-void AP_RangeFinder_MAVLink::handle_msg(mavlink_message_t *msg)
+*/void AP_RangeFinder_MAVLink::handle_msg(mavlink_message_t *msg)
 {
     mavlink_distance_sensor_t packet;
     mavlink_msg_distance_sensor_decode(msg, &packet);
 
     // only accept distances for downward facing sensors
```

Comment:
This comment would go within the `if`.  Outside it should read something like, `// check distance is from downward-facing sensor`